### PR TITLE
CEDS 1638 1639 fixes

### PR DESCRIPTION
**Pull request by MaciejRewera**
This PR consists of changes in styles and content for ERS and EMR notifications presentation.

### CODE REVIEW

File: app/assets/stylesheets/partials/_typography.scss [hunk 1]
```diff
@@ -35,3 +34,20 @@ p.font-xsmall{
...
+====================================
+Utility bold
+Pulled from GDS elements typography
+====================================
+*/
+.bold {
+  font-weight: 700;
+}
+label > .bold {
+  display: block; /*temp fix until form story can fix label structure*/
```
Comment (ed0906) [inline app/assets/stylesheets/partials/_typography.scss]: I think I remember this causing issues in the Dec Service, what's it used for?

**Comment by PatrykRudnicki** [comment]
Please run Scala formatter, PR build failed on this

File: app/models/viewmodels/notificationspage/converters/ERSResponseConverter.scala [hunk 2]
```diff
@@ -40,44 +40,44 @@ class ERSResponseConverter @Inject()(decoder: Decoder, dateTimeFormatter: DateTi
...
-    soeCodeExplanationText.map(explanation => paragraph(s"$SoeCodeHeader $explanation"))
+    soeCodeExplanationText.map(explanation => code_explanation(SoeCodeHeader, explanation))
   }
 
-  private def buildIcsCodeExplanation(icsCode: String)(implicit messages: Messages): Option[String] = {
+  private def buildIcsCodeExplanation(icsCode: String)(implicit messages: Messages): Option[Html] = {
     val icsCodeExplanationText = decoder.ics(icsCode).map(code => messages(code.contentKey))
 
-    icsCodeExplanationText.map(explanation => paragraph(explanation))
+    icsCodeExplanationText.map(explanation => paragraph(explanation)).map(Html(_))
```
Comment (bambuchaAdm) [inline app/models/viewmodels/notificationspage/converters/ERSResponseConverter.scala]: Maybe paragraph should return `Html` already ?

Comment (MaciejRewera) [reply app/models/viewmodels/notificationspage/converters/ERSResponseConverter.scala]: 👍

File: app/assets/stylesheets/partials/_typography.scss (hunk 1 above)
Comment (MaciejRewera) [reply app/assets/stylesheets/partials/_typography.scss]: I don't know the origin. I've checked whole project for usages and cannot find any. Might be, that it is some "forgotten relict from the past".
I'll remove it then.

**Pull request merged**